### PR TITLE
1.19 blog post

### PR DESCRIPTION
**Pull request by duailibe**
ETA: 9/25

### CODE REVIEW

**Comment by lydell** [comment]
Looks like we forgot to add a changelog entry in #6157 – we need to add that to the blog post.

**Comment by duailibe** [comment]
I'll add, thanks @lydell

**Comment by fisker** [comment]
Should we mention `parser-postcss.js` is now minified? #6395

File: website/blog/2019-09-16-1.19.0.md [hunk 1]
```diff
@@ -0,0 +1,604 @@
...
+    Foo
+  {{/if}}
+  {{! Bar }}
+  {{#if @bar}}
+    Bar
+  {{/if}}
+</div>
+```
+
+### Handlebars: Improve comment formatting ([#6234] by [@gavinjoyce])
```
Comment (sosukesuzuki) [inline website/blog/2019-09-16-1.19.0.md]: Should this be like below?
```suggestion
#### Improve comment formatting ([#6234] by [@gavinjoyce])
```

**Comment by kachkaev** [comment]
Would it be possible to wait for https://github.com/prettier/prettier/pull/6519 or can it be later released as 1.19.x?

File: website/blog/2019-09-25-1.19.0.md [hunk 2]
```diff
@@ -0,0 +1,731 @@
+---
+author: Lucas Duailibe (@duailibe)
+authorURL: https://github.com/duailibe
+title: "Prettier 1.19: Vue script and style indentation, and skipping unchanged files"
+---
+
+This release includes as asual lots of fixes and improvements, but most importantly includes a feature very requested by the Vue community to indent `script` and `style` tags contents and a flag `--only-changed` to speed up Prettier CLI runs.
```
Comment (nikulis) [inline website/blog/2019-09-25-1.19.0.md]: misspelling
```suggestion
This release includes as usual lots of fixes and improvements, but most importantly includes a feature much requested by the Vue community to indent `script` and `style` tags' contents and a flag `--only-changed` to speed up Prettier CLI runs.
```

File: website/blog/2019-09-25-1.19.0.md [hunk 3]
```diff
@@ -0,0 +1,731 @@
...
+
+// Output (Prettier 1.19)
+foo // comment
+`
+`;
+```
+
+#### Fix moving comments in function calls like `useEffect` second argument ([#6270] by [@sosukesuzuki])
+
+This fixes a bug that was affecting function calls that have a arrow function as first argument and an array expression as second argument, such as the common React's `useEffect`. A comment in its own line before the second argument would be moved to the line above.
```
Comment (nikulis) [inline website/blog/2019-09-25-1.19.0.md]:
```suggestion
This fixes a bug that was affecting function calls that have an arrow function as first argument and an array expression as second argument, such as the common React's `useEffect`. A comment in its own line before the second argument would be moved to the line above.
```

File: website/blog/2019-09-25-1.19.0.md [hunk 4]
```diff
@@ -0,0 +1,731 @@
...
+<p>
+  Some escaped characters: &lt; &gt; &amp;
+</p>
+```
+
+### Vue
+
+#### Don't break the template element included in a line shorter than print width ([#6284] by [@sosukesuzuki])
+
+Previously, even if the line length is shorter than print-width is Prettier breaks the line with a template element.
```
Comment (nikulis) [inline website/blog/2019-09-25-1.19.0.md]:
```suggestion
Previously, even if the line length is shorter than print-width, Prettier broke the line with a template element.
```

**Comment by duailibe** [comment]
I think any other updates should come in 1.20

**Comment by evilebottnawi** [comment]
@duailibe i disagree, all issues/PRs marked as 1.19 should be competed (some PRs are very important)

**Comment by duailibe** [comment]
@evilebottnawi Yes, what I meant was I don't think all the issues and PRs in the milestone need to be there and the release should not depend on that

**Comment by evilebottnawi** [comment]
@duailibe i think we should resolve this issue for `1.19`, some of them is very important

**Comment by lipis** [comment]
All set.. we just need to update the blogpost, and there are not leftovers in the 1.19 milestone.

**Comment by lydell** [comment]
I could finish this up this weekend, and probably do the release, too. Is it good to release on a weekend?

@duailibe Are you going to finish this, or should I take over?

@evilebottnawi @thorn0 @sosukesuzuki @fisker @brodybits Any comments?

**Comment by thorn0** [comment]
@lydell Don't forget that CHANGELOG.unreleased.md has been a little refined since this PR was created.

**Comment by lydell** [comment]
Yeah, I was thinking of starting from scratch from it.

**Comment by lydell** [comment]
Closing in favor of #6787.

**Pull request closed without merging**